miio/yeelight_dual_switch.py

```diff
@@ -0,0 +1,233 @@
+        click.argument("switch_1_status", type=bool),
+        default_output=format_output("Set First Switch Status to: {switch_1_status}"),
+    )
+    def set_switch_1_status(self, switch_1_status: bool):
+        """Set First Switch Status."""
+        return self.set_property("switch_1_status", switch_1_status)
+
+    @command(
+        click.argument("switch_1_default_state", type=EnumType(Mode)),
+        default_output=format_output(
+            "Set First Switch Default State to: {switch_1_default_state}"
+        ),
+    )
+    def set_switch_1_default_state(self, switch_1_default_state: bool):
+        """Set First Switch Default State."""
+        return self.set_property("switch_1_default_state", switch_1_default_state.value)
+
+    @command(
+        click.argument("switch_1_delay", type=int),
+        default_output=format_output("Set First switch delay to {switch_1_delay}%"),
+    )
+    def set_switch_1_delay(self, delay: int):
+        """Set First switch delay, should be between -1 to 43200"""
+        if delay < -1 or delay > 43200:
+            raise YeelightDualControlModuleException("Invalid switch delay: %s" % delay)
+        return self.set_property("switch_1_delay", delay)
+
+    @command(
+        click.argument("switch_2_status", type=bool),
+        default_output=format_output("Set Second Switch Status to: {switch_2_status}"),
+    )
+    def set_switch_2_status(self, switch_2_status: bool):
+        """Set First Switch Status."""
+        return self.set_property("switch_2_status", switch_2_status)
+
+    @command(
+        click.argument("switch_2_default_state", type=EnumType(Mode)),
+        default_output=format_output(
+            "Set Second Switch Default State to: {switch_2_default_state}"
+        ),
+    )
+    def set_switch_2_default_state(self, switch_2_default_state: bool):
+        """Set Second Switch Default State."""
+        return self.set_property("switch_1_default_state", switch_2_default_state.value)
+
+    @command(
+        click.argument("switch_2_delay", type=int),
+        default_output=format_output("Set Second switch delay to {switch_2_delay}%"),
+    )
+    def set_switch_2_delay(self, delay: int):
+        """Set Second switch delay, should be between -1 to 43200"""
+        if delay < -1 or delay > 43200:
+            raise YeelightDualControlModuleException("Invalid switch delay: %s" % delay)
+        return self.set_property("switch_2_delay", delay)
+
+    @command(
+        click.argument("flex_mode", type=bool),
+        default_output=format_output("Set Flex Mode to: {flex_mode}"),
+    )
+    def set_flex_mode(self, flex_mode: bool):
+        """Set Flex Mode."""
+        return self.set_property("flex_mode", (0, 1)[flex_mode])
+
+    @command(
+        click.argument("rc_list_for_del", type=str),
+        default_output=format_output("Delete remote control: {rc_list_for_del}"),
+    )
+    def delete_rc(self, rc_list_for_del: str):
+        """Delete Remote Control"""
```

Comment:
Could you add an example on how to use this, especially concerning on what/how the input value should look like?

Reply:
not sure (. I don't have remote control for testing. Will try to found some info about it

Reply:
need to pass rc MAC as a string.

Reply:
Okay, I'd then rename the variable to `mac` and set the docstring to `Delete remote control by mac` or so.